src/Microsoft.Android.Runtime.NativeAOT/Android.Runtime.NativeAOT/TypeMapping.cs

```diff
@@ -0,0 +1,68 @@
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.IO.Hashing;
+using System.Runtime.InteropServices;
+using System.Text;
+
+using Android.Runtime;
+
+namespace Microsoft.Android.Runtime;
+
+internal static class TypeMapping
+{
+	// Initialization generated by src/Microsoft.Android.Sdk.ILLink/TypeMappingStep.cs
+	private static readonly ulong[] s_hashes = null!;
```

Comment:
Can this be a property that returns ReadOnlySpan to avoid the array allocation?

Reply:
Thanks for the suggestion. I modified the code to avoid the allocation.